crowbar_framework/app/assets/stylesheets/content/header.scss

```diff
@@ -1,35 +1,21 @@
 .navbar-inverse {
   .navbar-brand {
-    padding: 4px 15px 0;
+    padding: 0 15px 0;
 
     .title {
       display: block;
       text-indent: -100000px;
-      background: transparent image-url('logos/crowbar.png') no-repeat;
-      width: 244px;
-      height: 42px;
+      background: transparent image-url("logo.png") no-repeat;
```

Comment:
Color literals like `transparent` should only be used in variable declarations; they should be referred to via variable everywhere else.